pyproject.toml

```diff
@@ -31,28 +31,16 @@ whitenoise = "^6.8.2"
 boto3 = "^1.34.162"
 pydantic = "^2.10.4"
 datamodel-code-generator = "^0.26.4"
-langchain = "^0.2"
-hdbscan = "^0.8.40"
-sentence-transformers = "^3.3.1"
-bertopic = "^0.16.4"
-scikit-learn = "^1.6.0"
-umap-learn = "^0.5.7"
-scipy = "^1.14.1"
-numpy = "^1.26.4"
 django-types = "^0.20.0"
 django-waffle = "^4.2.0"
 django-magic-link = "^1.0.0"
 django-crispy-forms = "^2.3"
 crispy-forms-gds = "^0.3.2"
 django-gov-notify = { git = "https://github.com/i-dot-ai/django-gov-notify", branch = "support-python-3-12" }
 sentry-sdk = { extras = ["redis", "rq"], version = "^2.19.2" }
-torch = "~2.2"
```

Comment:
I think you should be able to remove this from the ignore list in `.github/dependabot.yml`

Reply:
Well spotted - now removed.